packages/config/utils/utils.js

```diff
@@ -65,6 +87,97 @@ const mapContractToNetworkAddress = network => contracts => {
   }, {})
 }
 
+/**
+ *
+ * @param{string} network
+ * @param{FlowJSONv2.accounts|FlowJSONv2.contracts} objectPossiblyKeyedByNetwork
+ * @returns {Object} { "HelloWorld": "0x123" }
+ */
+const collapseByNetwork = (network, objectPossiblyKeyedByNetwork) => {
+  const result = {}
+  for (const [name, value] of Object.entries(objectPossiblyKeyedByNetwork)) {
+    if (typeof value === "string") {
+      result[name] = value
+      continue
+    }
+
+    if (value[network]) {
+      result[name] = value[network]
+    }
+  }
+
+  return result
+}
+
+/**
+ * @param{string} network
+ * @returns {Object} { "HelloWorld": "0x123" }
+ */
+const resolveV2ContractAliases = network => flowJSON => {
+  const collapsedAccounts = collapseByNetwork(network, flowJSON.accounts)
+  const collapsedContracts = collapseByNetwork(network, flowJSON.contracts)
+
+  const result = {}
+  for (const [contractName, contractValue] of Object.entries(
+    collapsedContracts
+  )) {
+    if (collapsedAccounts[contractValue]) {
+      result[`contracts.${contractName}`] = collapsedAccounts[contractValue]
+      continue
+    }
+
+    /**
+     * For simplicity, will assume we have an emulator address if the alias
+     * does not exist, and we are on the emulator network.
+     */
+    if (network === "emulator") {
+      result[`contracts.${contractName}`] = contractValue
+    }
+  }
+
+  return result
+}
+
+/**
+ * Take in a flow.json and return true if the schema is v1
+ * @param{FlowJSONv1|FlowJSONv2} obj
+ * @returns{boolean}
+ * @see https://github.com/onflow/flow-cli/issues/711
+ */
+const isFlowSchemaV1 = obj => {
+  // Deployments only exists in v1
+  if (obj.deployments) return true
```

Comment:
for now just return true because there is no v2 yet in the wild.

Reply:
Copy that. Given this will be released potentially, should I still return true?